src/Components/Navigation/Footer.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import './Footer.css';
+import { NavLink } from 'react-router-dom';
+import Logo from '../../Assets/Images/TCK PNG Logo.png';
+import facebookLogo from '../../Assets/Images/facebook-footer-logo.svg';
+import instagramLogo from '../../Assets/Images/instagram-footer-logo.svg';
+import twitterLogo from '../../Assets/Images/twitter-footer-logo.svg';
+
+function Footer() {
+  return (
+    <div className="footer-wrapper">
+      <NavLink
+        to="/"
+        className="tck-logo-wrapper"
+      >
+        <img src={Logo} alt="The Conscious Kid Logo" className="tck-logo-img" />
+      </NavLink>
+      <div className="menu-wrapper">
+        <p className="menu-title">
+          Menu
+        </p>
+        <NavLink
+          to="/"
+          className="menu-options"
+        >
+          Home
+        </NavLink>
+        <NavLink
+          to="/collection/init"
+          className="menu-options"
+        >
+          Collections
+        </NavLink>
+        <NavLink
+          to="/quiz"
+          className="menu-options"
+        >
+          Book Recommendation Quiz
+        </NavLink>
+        <NavLink
+          to="/dictionary"
+          className="menu-options"
+        >
+          Racial Literacy Dictionary
+        </NavLink>
+      </div>
+      <div className="resources-wrapper">
+        <p className="resources-title">
+          More from The Conscious Kid
+        </p>
+        <p className="resources-text">Resources</p>
```

Comment:
The resources and socials have specific URLs they must link to that I listed on the sprint card! Add those when you have time